scripts/qesap/qesap.py

```diff
@@ -5,6 +5,7 @@
 import os
 import argparse
 import logging
+import select
```

Comment:
I know about that warning but due to the content, the purpose and complicate nature of the code that need it, I like to keep the import there, hoping in the future to drop them all

Reply:
> I know about that warning but due to the content, the purpose and complicate nature of the code that need it, I like to keep the import there, hoping in the future to drop them all

In this case, Pylint complaints with unused imports.  If we want to keep a near-perfect score with Pylint we should drop these.  We can add them when we really need them after all...

Reply:
I propose to use local pyling suppression 
```
import select # pylint: disable=abstract-class-instantiated
```